BusBoard/BusBoard/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using RestSharp;
+using Newtonsoft.Json.Linq;
+
+namespace BusBoard
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Enter a bus stop ID: ");
+                var stopId = Console.ReadLine();
+
+                var tflApiUrl = "https://api.tfl.gov.uk";
+
+                var request = new RestRequest($"StopPoint/{stopId}/Arrivals");
+
+                var incomingBusPredictions =
+                    JsonConvert.DeserializeObject<List<IncomingBusPrediction>>(new RestClient(tflApiUrl).Get(request)
+                        .Content);
```

Comment:
I think you can do this with RestSharp rather than JsonConvert

```
var client = new RestClient(@"https://api.tfl.gov.uk");
var request = new RestRequest("StopPoint/{stopId}/Arrivals");
var incomingBusPredictions = client.Execute<List<IncomingBusPredictions>>(request).Data;
```